packages/blade/src/components/Carousel/Carousel.stories.tsx

```diff
@@ -577,4 +577,32 @@ const InteractiveCarouselTestimonialTemplate: StoryFn<typeof CarouselComponent>
 
 export const WithInteractiveCards = InteractiveCarouselTestimonialTemplate.bind({});
 
+export const WithPeek: StoryFn<typeof CarouselComponent> = (props) => {
+  return (
+    <Box margin="auto" padding="spacing.4" width="100%">
+      <Heading size="large">Peek Behavior - showPeek: true</Heading>
+      <Text marginTop="spacing.2" color="surface.text.gray.muted">
+        Active card is centered with adjacent cards peeking from the sides.
```

Comment:
Can you add more descriptive text here on what props need to be added to achive peek behavior, almost like a mini tutorial. 

Example: 

```jsx
 <Text>
          You can achive bleed by setting <Code size="medium">visibleItems</Code> to autofit &
          adding <Code size="medium">carouselItemWidth</Code> to be a fixed width (eg: 300px)
        </Text>
```